py/private/pytest.py.tmpl

```diff
@@ -32,6 +33,9 @@ if __name__ == "__main__":
         "no:cacheprovider",
     ]
 
+    if suite_name:
```

Comment:
Can this be moved within the conditional that sets `junit_xml`, as without that set this flag is moot?